README.md

```diff
@@ -78,12 +76,13 @@ main
 
 All Kubernetes-CSI repos are expected to switch to Prow. For details
 on what is enabled in Prow, see
-https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes-csi
+<https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes-csi>
 
 Test results for periodic jobs are visible in
-https://testgrid.k8s.io/sig-storage-csi-ci
+<https://testgrid.k8s.io/sig-storage-csi-ci>
```

Comment:
Same here. This check is likely to create a lot of churn in various repos if enabled.

Reply:
same as above.